src/pages/brokers/CreateBrokerPage.tsx

```diff
@@ -117,17 +117,17 @@ export default class CreateBrokerPage extends React.Component<IProps, IState> {
                         </div>
                         <div className="row">
                             <Form.Group controlId="code" className="col-md-3">
-                                <Form.Label>Nome</Form.Label>
+                                <Form.Label>Nome <span style={{color:"red"}}>*</span></Form.Label>
```

Comment:
instead of 
`<span style={{color:"red"}}>*</span>`
adopt the following
`<span className="text-danger">*</span>`


in general css in code should be avoided.